pkgs/by-name/rt/rtorrent/package.nix

```diff
@@ -20,13 +20,13 @@
 
 stdenv.mkDerivation rec {
   pname = "rakshasa-rtorrent";
-  version = "0.10.0-unstable-2024-12-15";
+  version = "0.15.0-unstable-2024-12-26";
```

Comment:
Thank you for updating this. [v0.15.1](https://github.com/rakshasa/rtorrent/releases/tag/v0.15.1) is out now, so please bump to that. Also, could you please remove the "unstable" suffix and a rev hash, this release is properly tagged upstream.